OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs

```diff
@@ -356,9 +356,16 @@ void RefreshServerList()
 					}
 				}
 
-				lanGames = lanGames.GroupBy(gs => gs.Address).Select(g => g.Last()).ToList();
+				var groupedLanGames = lanGames.GroupBy(gs => gs.Address).Select(g => g.Last());
+				if (groupedLanGames.Count() > 0)
```

Comment:
if (groupedLanGames.Count() > 0) could be replaced with if (groupedLanGames.Any()), but ok, I'll use exactly this.